meilix-artwork/debian/control

```diff
@@ -12,6 +12,12 @@ Depends: ${misc:Depends}, plymouth, plymouth-label
 Description: graphical boot animation and logger - meilix-logo theme
  This package contains the default meilix-logo theme.
 
+Package: plymouth-theme-meilix-logo2
```

Comment:
Adding a hyphen before 2 will make it more clear!!
Also you can edit the description slightly, 
All the other things looks ready to ship!!